crates/pera-core/src/authority.rs

```diff
@@ -803,6 +806,8 @@ pub struct AuthorityState {
     pub overload_info: AuthorityOverloadInfo,
 
     pub validator_tx_finalizer: Option<Arc<ValidatorTxFinalizer<NetworkAuthorityClient>>>,
+
+    mpc_state: Mutex<MPCService>,
```

Comment:
Is there a reason this `MPCService` object is called the `mpc_state` here? I would expect `mpc_service` as a name.